src/commands.h

```diff
@@ -44,26 +43,34 @@
 #define RS_ALIASDEL_IF_EX RS_CMD_WRITE_PREFIX "._ALIASDELIFX"  // for replica of support
 #define RS_ALIASUPDATE RS_CMD_WRITE_PREFIX ".ALIASUPDATE"
 
+// Suggestion commands are key-bounded, but gets redirection for OSS cluster.
+// So on standalone and enterprise, we need only the local `FT.` variations,
+// but on OSS cluster we need both the local and the remote variations, and
+// have the local variations with `_FT.` prefix.
+#define RS_SUGADD_CMD RS_CMD_WRITE_PREFIX ".SUGADD"
+#define RS_SUGGET_CMD RS_CMD_WRITE_PREFIX ".SUGGET"
```

Comment:
So the name `RS_CMD_WRITE_PREFIX` is not longer a good name

Reply:
I mean... For `SUGADD` and `SUGDEL` it was wrong previously, now it is wrong for `SUGGET` and `SUGLEN`. We can rename it, but for the other commands it's still true

Reply:
`SUGADD` and `SUGDEL` are writes commands, `SUGGET` and `SUGLEN` are not writes.

Reply:
Right, and although that's the case, `SUGADD` and `SUGDEL` had the `RS_CMD_READ_PREFIX` prefix previously